pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// InterPodAffinityArgs holds arguments used to configure the InterPodAffinity plugin.
+type InterPodAffinityArgs struct {
+	// HardPodAffinityWeight is the scoring weight for existing pods with a
+	// matching hard affinity to the incoming pod.
+	HardPodAffinityWeight *int32 `json:"hardPodAffinityWeight,omitempty"`
+}
+
+// NodeLabelArgs holds arguments that used to configure the NodeLabel plugin.
+type NodeLabelArgs struct {
+	// PresentLabels should be present for the node to be considered a fit for hosting the pod
+	PresentLabels []string `json:"presentLabels,omitempty"`
```

Comment:
Types in this files shouldn't have json tags

Reply:
So if we're going to use v1alpha2 types in the code of the plugins, this file can be deleted entirely as it wouldn't be used.